pinot-core/src/main/java/org/apache/pinot/core/accounting/CPUMemThreadLevelAccountingObjects.java

```diff
@@ -32,47 +30,49 @@
  */
 public class CPUMemThreadLevelAccountingObjects {
 
-  public static class StatsDigest {
-
-    // The current usage sampling for each thread
-    final long[] _currentStatsSample;
-    // The previous usage sampling for each thread
-    final long[] _lastStatSample;
-    // The aggregated usage sampling for the finished tasks of a (still) running queries
-    final HashMap<String, Long> _finishedTaskStatAggregator;
-
-    StatsDigest(int numThreads) {
-      _currentStatsSample = new long[numThreads];
-      _lastStatSample = new long[numThreads];
-      _finishedTaskStatAggregator = new HashMap<>();
-    }
-  }
-
   /**
-   * Entry to track the task execution status of a worker/runner given thread
+   * Entry to track the task execution status and usage stats of a Thread
```

Comment:
(nit) might want to add a note that this "Thread" can refer to a server worker thread, runner thread or the broker threads

Reply:
done